test/views/details/EmployerPayrollIdViewSpec.scala

```diff
@@ -32,9 +32,9 @@ class EmployerPayrollIdViewSpec extends ViewUnitTest {
 
   object Selectors {
     val paragraph0Selector = "#main-content > div > div > p.govuk-body:first-of-type"
```

Comment:
`paragraph2Selector`, `paragraph3Selector`, and `paragraph4Selector` select the 2nd, 3rd, and 4th paragraphs respectively. To match those the function that selects the first paragraph should be called `paragraph1Selector`. Could you rename it?